### PR TITLE
Convert from misk.logging to wisp.logging

### PR DESCRIPTION
**Pull request by chris-ryan-square**

### CODE REVIEW

**Comment by chris-ryan-square** [comment]
Whoops, this is going to fail.  Need a new release of Misk. :(

**Comment by shellderp** [comment]
> Whoops, this is going to fail. Need a new release of Misk. :(

it should be publishing staged artifacts on every master build, just go into sonatype and close the latest one.

**Comment by shellderp** [comment]
> > Whoops, this is going to fail. Need a new release of Misk. :(
> 
> it should be publishing staged artifacts on every master build, just go into sonatype and close the latest one.

oh I forgot we made it manual, gotta kick off the publish action first

**Review by lkerford — APPROVED** [review]

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -62,4 +62,5 @@ object Dependencies {
   val wireMoshiAdapter = "com.squareup.wire:wire-moshi-adapter:3.6.0"
   val wireRuntime = "com.squareup.wire:wire-runtime:3.6.0"
   val wireSchema = "com.squareup.wire:wire-schema:3.6.0"
+  val wispLogging = "com.squareup.misk:wisp-logging:2021.04.22-1619131614-2df6b0ee"
```
Comment (lkerford) [inline buildSrc/src/main/kotlin/Dependencies.kt:65]: I think this versions should be something like `0.17.0` maybe or something similar

Comment (chris-ryan-square) [reply buildSrc/src/main/kotlin/Dependencies.kt:65]: Yes, I'm in the process of some Misk changes, will publish a new version and fix this.

**Comment by chris-ryan-square** [comment]
Not required any more

**Pull request closed without merging**